pkg/controller/job/controller_test.go

```diff
@@ -43,7 +43,9 @@ func newJob(parallelism, completions int) *extensions.Job {
 		Spec: extensions.JobSpec{
 			Parallelism: &parallelism,
 			Completions: &completions,
-			Selector:    map[string]string{"foo": "bar"},
+			Selector: extensions.PodSelector{
+				MatchLabels: map[string]string{"foo": "bar"},
```

Comment:
Can you add to this test something that tests MatchExpressions? It seems that right now it only tests MatchLabels.